index.js

```diff
@@ -67,7 +67,13 @@ function PinoColada () {
     if (contentLength != null) output.push(formatBundleSize(contentLength))
     if (responseTime != null) output.push(formatLoadTime(responseTime))
 
-    return output.filter(noEmpty).join(' ')
+    var str = output.filter(noEmpty).join(' ')
+
+    // if error, output stack trace on new line
+    var err = obj.err
+    if (err && err.stack) str += nl + err.stack
```

Comment:
ok, so errors get outputed under https://github.com/lrlna/pino-colada/pull/13/files#diff-168726dbe96b3ce427e7fedce31bb0bcR97, and we do print of the stack trace for `fatal`. Are you currently logging those errors as `fatal`, or just `error`? I want to make sure it's not a bug with the way we handle it below, basically.